### PR TITLE
Update Roadshow 2018 map styles

### PR DESCRIPTION
**Pull request by bretmorris**
New layout:

screen shot 2018-03-20 at 3 13 39 pm

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -8,6 +8,9 @@ end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 
+[*.html]
+indent_size = 4
```
Comment (bpaduch) [inline .editorconfig]: Was this change intentional?

**Comment by bpaduch** [comment]
LGTM except for the one config change.  Will leave to @pbakaus  to approve.

**Comment by pbakaus** [comment]
Also, if you could sign the CLA, that would be much appreciated.

**Comment by bretmorris** [comment]
@bpaduch the html was using four spaces throughout but the editorconfig was trying to force to 2 spaces. I edited the config to reflect how the current html files are set up. happy to change if that's an issue, though.

**Comment by bpaduch** [comment]
@bretmorris - let's reset it back. Thanks!

**Comment by bpaduch** [comment]
@bretmorris - thanks.  Can you just sign the CLA, and we should be good.

**Comment by bretmorris** [comment]
`I signed it!`

**Comment by bpaduch** [comment]
Awesome. TY!
